Core xdr-retrieve-file different separator options (#45106)
demisto/content: Packs/ApiModules/ReleaseNotes/2_4_37.md

#### Scripts

##### CoreIRApiModule

> - Added the *paths_separator* argument to the file-retrieval command, allowing retrieval of files whose paths contain a comma by specifying a different delimiter.

Reviewer: ```suggestion
- Added the ***paths_separator*** argument to the file retrieval commands to support custom delimiters for paths containing commas.
```
